<ui version="4.0" stdsetdef="1" >
  <author/>
  <comment/>
  <exportmacro/>
  <class>dictClassChooser</class>
  <widget class="QDialog" name="dictClassChooser" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>310</width>
        <height>181</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Class chooser</string>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <layout class="QVBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <widget class="QListWidget" name="classListBox" >
                  <property name="sizePolicy" >
                    <sizepolicy>
                      <hsizetype>5</hsizetype>
                      <vsizetype>5</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer1" >
                  <property name="sizeHint" >
                    <size>
                      <width>20</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Minimum</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QLabel" name="commentLextLabel" >
                  <property name="sizePolicy" >
                    <sizepolicy>
                      <hsizetype>7</hsizetype>
                      <vsizetype>7</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="frameShape" >
                    <enum>QFrame::Box</enum>
                  </property>
                  <property name="margin" >
                    <number>5</number>
                  </property>
                  <property name="text" >
                    <string>ewtgagsdg
afdgagafdgadfgag</string>
                  </property>
                  <property name="alignment" >
                    <set>Qt::AlignTop</set>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <spacer name="spacer2" >
                  <property name="sizeHint" >
                    <size>
                      <width>40</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QPushButton" name="pushButton1" >
                  <property name="text" >
                    <string>Cancel</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer2_2" >
                  <property name="sizeHint" >
                    <size>
                      <width>50</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Horizontal</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <connections/>
</ui>
